test/unit/specs/store/ledger.spec.js

```diff
@@ -0,0 +1,382 @@
+import ledgerModule from "modules/ledger.js"
+import { App, comm_u2f } from "ledger-cosmos-js"
+
+describe(`Module: Ledger`, () => {
+  let module, state, actions, mutations
+
+  beforeEach(() => {
+    module = ledgerModule()
+    state = module.state
+    actions = module.actions
+    mutations = module.mutations
+  })
+
+  describe(`Mutations`, () => {
+    it(`sets the Ledger app`, async () => {
+      const comm = await comm_u2f.create_async(5, true)
+      const app = new App(comm)
+      mutations.setLedger(state, app)
+      expect(state.app).toMatchObject(app)
+    })
+
+    it(`sets the Ledger cosmos app version`, () => {
+      const version = { major: 0, minor: 1, patch: 1, test_mode: false }
+      mutations.setLedgerCosmosVersion(state, version)
+      expect(state.version).toMatchObject(version)
+    })
+
+    it(`sets the account public key`, () => {
+      const pubKey = Buffer.from([
```

Comment:
for another time: I think hex values would be more readable

Reply:
updated